### DISCUSSION TITLE
Question About Documentation

### DISCUSSION THREAD

**Discussion by matibonvin** [ldproxy/ldproxy#1281] [General]
Hi,

I'm new to Idproxy and trying to get started. I noticed the Creating APIs section of the documentation is marked as "coming soon." Do you know when it will be available, or if there are any other resources I can use in the meantime?

Thanks!

**Comment by cportele** [comment]
Hi @matibonvin, we do not have a clear estimate when that section will be added.

We maintain a repository with the configurations on https://demo.ldproxy.net/ including instructions to prepare the data. This probably the best starting point with example configurations that you can use as a starting point to experiment. The repository is at https://github.com/ldproxy/demo.
